### ISSUE TITLE
adding an entity to config locked HA

### ISSUE THREAD

**Issue by Mariusthvdb** [alexarch21/history-explorer-card#2]
as per request in followup of https://community.home-assistant.io/t/new-interactive-history-explorer-custom-card/369450/26?u=mariusthvdb

my HA instance locked, auto- restarted and logged:
```
2022-01-12 14:13:31 WARNING (MainThread) [homeassistant.core] Timed out waiting for shutdown stage 1 to complete, the shutdown will continue
2022-01-12 14:13:33 WARNING (Thread-106) [homeassistant.util.executor] Thread[SyncWorker_2] is still running at shutdown: File "/usr/local/lib/python3.9/threading.py", line 930, in _bootstrap
    self._bootstrap_inner()
  File "/usr/local/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 75, in _worker
    work_item = work_queue.get(block=True)
```

not sure if this is because of the resource, but it was happening upon adding a single (and first) entity to the card.....

Home Assistant 2022.2.0.dev20220112

yaml:
```
  - title: History
    icon: mdi:history
    path: history
    type: panel
    cards:
     - type: custom:history-explorer-card
```

**Comment by alexarch21** [comment]
I see that you're using a beta / dev build of HA. Do you have a system running a stable build and if so, do you also experience this issue on that stable build ? The error above is something in HA itself crashing (although probably triggered by the card).

**Comment by Mariusthvdb** [comment]
still on the dev install, I installed the new update and use the
```
- title: History
  icon: mdi:history
  path: history
  type: panel
  cards:
    - type: custom:history-explorer-card
      showUnavailable: false
      recordedEntitiesOnly: true
```

options. I can see the entities being filtered in the dropdown, so thats nice! thanks for that. 


after some initial issues of not seeing a graph after clicking the + sign (not sure what happens there) I got this after selecting the entities nicely:

<img width="1207" alt="Schermafbeelding 2022-01-17 om 10 18 07" src="https://user-images.githubusercontent.com/33354141/149741914-0af6160e-7732-4aa5-a34e-25bc1683ebea.png">

didnt yet lock up either ;-)
